packages/nimble-components/src/table-column/enum-text/tests/table-column-enum-text.spec.ts

```diff
@@ -359,7 +366,7 @@ describe('TableColumnEnumText', () => {
         async function setupInvalidMappings(): Promise<ModelInvalidMappingsFixture<Table<SimpleTableRecord>>> {
             const source = new ModelInvalidMappings();
             const result = await fixture<Table<SimpleTableRecord>>(html<ModelInvalidMappings>`
-                <${tableTag} style="width: 700px">
+                <${tableTag} ${ref('table')} style="width: 700px">
```

Comment:
The typing in these tests was a little off, so I was mostly just trying to make the compiler happy. The root of the problem is that the tests were assigning an `ModelInvalidMappings` instance to a variable typed as `Model`, so they needed to be identical. However, since they are identical, I've now gotten rid of `ModelInvalidMappings` and `ModelInvalidMappingsFixture` in favor of using `Model` and `ModelFixture` in the invalid setup (since ultimately they were already implicitly required to be the same).